addons/payment/migrations/11.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,48 @@
+---Fields in module 'payment'---
+payment      / payment.acquirer         / auto_confirm (selection)      : DEL required: required, selection_keys: ['authorize', 'confirm_so', 'generate_and_pay_invoice', 'none'], req_default: function
+payment      / payment.acquirer         / capture_manually (boolean)    : NEW 
+payment      / payment.acquirer         / country_ids (many2many)       : NEW relation: res.country
+payment      / payment.acquirer         / payment_flow (selection)      : NEW required: required, selection_keys: ['form', 's2s'], req_default: function
+payment      / payment.acquirer         / payment_icon_ids (many2many)  : NEW relation: payment.icon
+payment      / payment.acquirer         / specific_countries (boolean)  : NEW 
+payment      / payment.icon             / acquirer_ids (many2many)      : NEW relation: payment.acquirer
+payment      / payment.icon             / image (binary)                : NEW attachment: True
+payment      / payment.icon             / image_payment_form (binary)   : NEW attachment: True
+payment      / payment.icon             / name (char)                   : NEW 
+payment      / payment.token            / verified (boolean)            : NEW 
+payment      / payment.transaction      / callback_eval (char)          : DEL 
```

Comment:
I have been searching in current 11.0 code (not OU one, which is outdated), possible uses of new fields that can be referred to this old field in previous version, but I can't find any. Did you figure out any possible translation needed for this field?